src/main/java/ca/nrc/cadc/beacon/web/restlet/VOSpaceStatusService.java

```diff
@@ -141,16 +142,22 @@ public Status toStatus(final Throwable throwable, final Request request, final R
             status = (cause == null)
                      ? super.toStatus(throwable, request, response) : toStatus(cause, request, response);
         } else if (throwable instanceof IllegalArgumentException) {
-            status = new Status(Status.CLIENT_ERROR_BAD_REQUEST.getCode(), throwable.getMessage(),
-                                throwable.getMessage());
+            // Prune out any CR or LF that might come from web services, or they'll
+            // get converted to a helpful message about how they need to be removed
+            // further up the chain in the restlet code.
+            String thrownMessage = throwable.getMessage();
+            if (thrownMessage.contains("\n")) {
+                thrownMessage = thrownMessage.replace("\n", "");
```

Comment:
You can just put this on a single line:
`throwable.getMessage().replace("\n", "");`

Reply:
done